opba-facade-protocol-api-shared/src/main/java/de/adorsys/opba/protocol/api/services/EncryptionService.java

```diff
@@ -0,0 +1,16 @@
+package de.adorsys.opba.protocol.api.services;
+
+import javax.crypto.SecretKey;
+
+public interface EncryptionService {
```

Comment:
EncryptionService interface for BankProtocol requires 
EncryptionService.decrypt(byte[] data)
EncryptionService.encrypt(byte[] data)
without knowledge of secret key

Reply:
How it will know who's key to use?

Reply:
ok encryptionService has RequestScope